draft-ietf-quic-transport.md

```diff
@@ -830,25 +830,26 @@ STREAM frames is used to carry other frame-specific flags.  For all
 other frames, the Frame Type byte simply identifies the frame.  These frames are
 explained in more detail as they are referenced later in the document.
 
-| Type Value  | Frame Type Name   | Definition                  |
-|:------------|:------------------|:----------------------------|
-| 0x00        | PADDING           | {{frame-padding}}           |
-| 0x01        | RST_STREAM        | {{frame-rst-stream}}        |
-| 0x02        | CONNECTION_CLOSE  | {{frame-connection-close}}  |
-| 0x03        | APPLICATION_CLOSE | {{frame-application-close}} |
-| 0x04        | MAX_DATA          | {{frame-max-data}}          |
-| 0x05        | MAX_STREAM_DATA   | {{frame-max-stream-data}}   |
-| 0x06        | MAX_STREAM_ID     | {{frame-max-stream-id}}     |
-| 0x07        | PING              | {{frame-ping}}              |
-| 0x08        | BLOCKED           | {{frame-blocked}}           |
-| 0x09        | STREAM_BLOCKED    | {{frame-stream-blocked}}    |
-| 0x0a        | STREAM_ID_BLOCKED | {{frame-stream-id-blocked}} |
-| 0x0b        | NEW_CONNECTION_ID | {{frame-new-connection-id}} |
-| 0x0c        | STOP_SENDING      | {{frame-stop-sending}}      |
-| 0x0d        | ACK               | {{frame-ack}}               |
-| 0x0e        | PATH_CHALLENGE    | {{frame-path-challenge}}    |
-| 0x0f        | PATH_RESPONSE     | {{frame-path-response}}     |
-| 0x10 - 0x17 | STREAM            | {{frame-stream}}            |
+| Type Value  | Frame Type Name    | Definition                  |
+|:------------|:-------------------|:----------------------------|
+| 0x00        | PADDING            | {{frame-padding}}           |
+| 0x01        | RST_STREAM         | {{frame-rst-stream}}        |
+| 0x02        | CONNECTION_CLOSE   | {{frame-connection-close}}  |
+| 0x03        | APPLICATION_CLOSE  | {{frame-application-close}} |
+| 0x04        | MAX_DATA           | {{frame-max-data}}          |
+| 0x05        | MAX_STREAM_DATA    | {{frame-max-stream-data}}   |
+| 0x06        | MAX_STREAM_ID      | {{frame-max-stream-id}}     |
+| 0x07        | PING               | {{frame-ping}}              |
+| 0x08        | BLOCKED            | {{frame-blocked}}           |
+| 0x09        | STREAM_BLOCKED     | {{frame-stream-blocked}}    |
+| 0x0a        | STREAM_ID_BLOCKED  | {{frame-stream-id-blocked}} |
+| 0x0b        | NEW_CONNECTION_ID  | {{frame-new-connection-id}} |
+| 0x0c        | STOP_SENDING       | {{frame-stop-sending}}      |
+| 0x0d        | ACK                | {{frame-ack}}               |
+| 0x0e        | PATH_CHALLENGE     | {{frame-path-challenge}}    |
+| 0x0f        | PATH_RESPONSE      | {{frame-path-response}}     |
+| 0x10 - 0x17 | STREAM             | {{frame-stream}}            |
+| 0x80 - 0xff | (extension frames) | {{frame-extension}}         |
```

Comment:
Maybe save `0xFF` for a "super-extension" to allow for more than 128 extended frames?

Reply:
As opposed to...say 0x7f, which remains unassigned?  I'd say that if we haven't changed the QUIC version number by that point, something went (badly) wrong.